exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/_log_exporter/__init__.py

```diff
@@ -82,6 +88,20 @@ def __init__(
             OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE,
             environ.get(OTEL_EXPORTER_OTLP_CERTIFICATE, True),
         )
+        self._client_key_file = client_key_file or environ.get(
```

Comment:
Looks like these new fields can just be local variables (ditto the other signals). If they are fields instead of locals because they are used by tests, would it be helpful to readers to add a comment?

Reply:
Good point, added a comment.